src/pages/home/report/ReportActionCompose/ReportActionCompose.tsx

```diff
@@ -342,6 +342,7 @@ function ReportActionCompose({
         [],
     );
 
+    const isChatRoom = useMemo(() => ReportUtils.isGroupPolicy(report), [report]);
```

Comment:
```suggestion
    const isGroupPolicyReport = useMemo(() => ReportUtils.isGroupPolicy(report), [report]);
```

I think this won't just apply to chat rooms, but any group policy report (e.g. an expense report). Which is what we want, but we should have the variable name properly reflect that.